index.js

```diff
@@ -2,11 +2,8 @@ const sass = require('node-sass')
 
 module.exports = (css, settings) => {
   const cssWithPlaceholders = css
-    .replace(/:\s*%%styled-jsx-placeholder-(\d+)%%/g, (_, id) =>
-      `: styled-jsx-placeholder-${id}`
-    )
-    .replace(/%%styled-jsx-placeholder-(\d+)%%\s*{/g, (_, id) =>
-      `styled-jsx-placeholder-${id} {`
+    .replace(/%%styled-jsx-placeholder-(\d+)%%(\w*\s*[);{])/g, (_, id, p1) =>
```

Comment:
can you add a comment with some examples of matches?

Reply:
Do you mean something like:
````javascript
/**
It matches:
color: %%styled-jsx-placeholder-0%%;
border: 1px solid %%styled-jsx-placeholder-0%%;
border: 1px solid %%styled-jsx-placeholder-0%%em;
@media %%styled-jsx-placeholder-0%% {
@media (min-height: %%styled-jsx-placeholder-0%%)
**/
````
?

I was thinking would it be better to make more tests for just a single scenario like examples above...?

Reply:
What is the `[` for in the regex? 
You can have multiple `replace` but in the future I will try to make it easier to do these replacement for plugin authors.

Reply:
`[);{]` is a character-class, it matches one of the characters in brackets, in this case - ) ; {
I remove `replace` because it wasn't needed.

Reply:
@giuseppeg is there anything I should do?

Reply:
oh sorry i didn't think about the character class haha. No I am out now I will merge and release tomorrow morning.

Reply:
Anything else required for this to be merged in?